compiler/rustc_mir_transform/src/required_consts.rs

```diff
@@ -17,7 +17,7 @@ impl<'tcx> Visitor<'tcx> for RequiredConstsVisitor<'_, 'tcx> {
         let literal = constant.literal;
         match literal {
             ConstantKind::Ty(c) => match c.kind() {
-                ConstKind::Param(_) => {}
+                ConstKind::Param(_) | ConstKind::Error(_) => {}
                 _ => bug!("only ConstKind::Param should be encountered here, got {:#?}", c),
```

Comment:
I don't remotely know how this works, but shouldn't this be changed too, since the above now allows another kind?

Reply:
It's just an error message, so I don't think it really needs to be changed -- Error isn't really *expected* here, we just silently swallow it.